lives-of-primitive-quakers/modernized/06-chapter-5.adoc

```diff
@@ -66,8 +66,7 @@ and the former proceedings against, him,
 and how unequal it was that they waived the other
 statutes lately made against nonconformists,
 and prosecuted him upon a statute formerly made against Catholic recusants +++[+++
-those who would not recongnize or submit to the Church of England]
-. He said,
+those who would not recognize or submit to the Church of England]. He said,
```

Comment:
You were right to move the period up to the line before, but I think the "He said," should be on the next line, since it starts a sentence.  You can probably just leave it as part of the next line, I don't think it will be too long.